libdevcore/CommonIO.h

```diff
@@ -48,6 +48,10 @@ std::string contentsString(std::string const& _file);
 /// Retrieve and returns the allocated contents of the given file. If the file doesn't exist or isn't readable, returns nullptr. Don't forget to delete [] when finished.
 bytesRef contentsNew(std::string const& _file, bytesRef _dest = bytesRef());
```

Comment:
this does the same as `contentsToBuffer` when passed a second param, except that it's size-safe.